samplesheets/views_api.py

```diff
@@ -52,6 +61,7 @@
 MD5_RE = re.compile(r'([a-fA-F\d]{32})')
 APP_NAME = 'samplesheets'
 IRODS_QUERY_ERROR_MSG = 'Exception querying iRODS objects'
+IRODS_EX_MSG = ' iRODS data request failed'
```

Comment:
Same thing as with the trailing special characters, we should not have leading spaces or special characters in a reusable string constant.